streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionVertex.java

```diff
@@ -27,16 +33,26 @@
    * Unique name generated by vertex name and index for execution vertex.
    */
   private final String vertexName;
+  /**
+   * Resources used by ExecutionVertex.
+   */
+  private final Map<String, Double> resources;
+
+  private final Map<String, String> jobConfig;
```

Comment:
I think `ExecutionVertex` should only need ExecutionConfig. `jobConfig ` should be accessed using other ways such as `getRuntimeContext().getJobConfig`?

Reply:
Use ```RuntimeContext``` replace to ```jobConf```.